### ISSUE TITLE
Django user permissions

### ISSUE THREAD

**Issue by odscjames** [OpenDataServices/json-data-ferret#9]
So only some users can access each admin UI (core app and example app, different permission for both)

**Comment by aayushi-droid** [comment]
Yes you can give permission to specific used based on their Email account.
Like Give All accress if user.email.endswith("@example.com")
I am also working on same soon.

**Comment by odscjames** [comment]
Thanks, but I was intending to use https://docs.djangoproject.com/en/3.0/topics/auth/default/#the-permission-required-decorator 

    @permission_required('polls.add_choice')

That gives us more fine grained control on granting access to individual users. We don't have many users (at least to start with) so we don't need a rule based approach.

**Comment by odscjames** [comment]
Done for main app, maybe do for example app later
